### ISSUE TITLE
Soundcore Space A40/A3936 Support

### ISSUE THREAD

**Issue by gmallios** [gmallios/SoundcoreManager#19]
Current status of Soundcore Space A40 (A3936) support:
- [ ] Connection
- [ ] Fetch device status 
- [ ] ANC
- [ ] EQ
- [ ] UI
- [ ] Response Checksum Validation

**Comment by RodoMa92** [comment]
I should get the required logs shortly. Pulling them from android is a pain but from a look in wireshark it should be good to go.

**Comment by RodoMa92** [comment]
Yeah, after taking a peek in the app it is of course heavily obfuscated mainly in the part that are sensible to create a cross compatible app (communication protocols). I was searching for a generic library to control various vendors of audio gear with a Decky plugin app for Linux, and this looks a great project to start with for SoundCore hardware (besides the issue with the RFCOMM library on other platform, but I can take a look at that and see if I can replace it with something else).

~Anyway, here are the logs while interacting with the app.~

Removed since OpenSCQ30 now has implemented support for it, therefore I don't really need this anymore and I would prefer to not leave mac addresses around the web if not needed.
